fluid/fluid_gui.ui

```diff
@@ -50,7 +50,11 @@
         <string/>
        </property>
        <property name="text">
-        <string notr="true" extracomment="gets replaced with arrowUp icon">↑</string>
+        <string notr="true"/>
+       </property>
+       <property name="icon">
+        <iconset resource="../mscore/musescore.qrc">
+         <normaloff>:/data/icons/arrow_up.svg</normaloff>:/data/icons/arrow_up.svg</iconset>
```

Comment:
For the record, it was done this way to support retina display. In CPP it was working, and in UI file it wasn't, the icon was bigger than the button. I just tested and it seems Qt 5.6 is working better. So I will merge the PR.